### PR TITLE
[qute] Codelenses disappear once @CheckedTemplate is followed by parenthesis

### LINKED ISSUE #439: [qute] Codelenses disappear once @CheckedTemplate is followed by parenthesis
see 
Nov-24-2021 12-18-20

### PR DESCRIPTION
**Pull request by angelozerr**
[qute] Codelenses disappear once @CheckedTemplate is followed by parenthesis

Fixes #439

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
It should fix the issue. I set as draft because I need to write some tests again.

File: qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/internal/java/QuteErrorCode.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.qute.jdt.internal.java;
+
+public enum QuteErrorCode implements IQuteErrorCode {
+
+	NoTemplateMatching("No template matching the path {0} could be found for: {1}");
```
Comment (fbricon) [inline qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/internal/java/QuteErrorCode.java:16]: NoMatchingTemplate(...)

**Comment by angelozerr** [comment]
The fix was very simple, but I spend some time to write tests for document link, codelens and diagnostics.

File: qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/internal/java/QuteErrorCode.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.qute.jdt.internal.java;
+
+public enum QuteErrorCode implements IQuteErrorCode {
+
+	NoMatchingTemplate("No template matching the path {0} could be found for: {1}");
```
Comment (fbricon) [inline qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/internal/java/QuteErrorCode.java:16]: remove the `:`

**Comment by fbricon** [comment]
Feel free to merge once the NoMatchingTemplate message is fixed

File: qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/internal/java/QuteErrorCode.java (hunk 2 above)
Comment (angelozerr) [reply qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/internal/java/QuteErrorCode.java:16]: I did that, to follow the same message than Quarkus.

Comment (fbricon) [reply qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/internal/java/QuteErrorCode.java:16]: ok then

**Comment by fbricon** [comment]
Thanks @angelozerr!

**Pull request merged**